saas/templates/saas/base_dashboard.html

```diff
@@ -25,62 +89,7 @@
     $interpolateProvider.startSymbol('[[');
     $interpolateProvider.endSymbol(']]');
 }]);
-reopenSaasApp.constant('settings', {
```

Comment:
The whole point of doing ``reopenSaasApp.constant('settings', {...`` is that we do not rely on a global variables. Globals in Javascript, especially in libraries, create lots of integration problems down the line.